docs/samples/calling/style.css

```diff
@@ -329,3 +328,48 @@ select.ServiceData {
 select.ServiceData:invalid {
   color: #555555d4;
 }
+
+h2 {
+  color: #0052bf;
+  font-size: 1.5rem;
+  width: 100%;
+}
+
+legend {
+  font-size: 1.15rem;
+  color: #1a73e8;
+}
+
+.collapsible {
+  cursor: pointer;
+}
+
+.section-content {
+  height: auto;
+  visibility: visible;
+}
+
+.collapsed {
+  height: 0;
+  visibility: hidden;
+}
+
+.text-color {
+  color: #555;
+}
+
+input[type="string"],
+select {
+  padding: 0 0.5rem;
+  height: 1.75rem;
+  border-radius: 4px;
```

Comment:
can we use rem value here as well ?

Reply:
We should not change border radius if the root font-size changes.